pages/search.js

```diff
@@ -1,41 +1,114 @@
 import PropTypes from "prop-types";
+import React, { useState } from "react";
 import Breadcrumbs from "../components/breadcrumbs";
 import {
   DataArea,
   DataItemLabel,
   DataItemValue,
   DataPanel,
 } from "../components/data-area";
+import { Button } from "../components/form-elements";
 import Notifications from "../components/notifications";
 import PagePreamble from "../components/page-preamble";
 import RegulomeVersionTag from "../components/regulome-version-tag";
+import NearbySNPsDrawing from "../components/snps-diagram";
 import errorObjectToProps from "../lib/errors";
 import FetchRequest from "../lib/fetch-request";
+import filterOverlappingPeaks from "../lib/filter-overlapping-peaks";
+import getSnpsInfo from "../lib/get-snps-info";
 import { getQueryStringFromServerQuery } from "../lib/query-utils";
 
+// Default number of populations to display for allele frequencies.
+const defaultDisplayCount = 3;
+
 export default function Search({ data }) {
-  return (
-    <>
-      <Breadcrumbs />
-      <RegulomeVersionTag />
-      <PagePreamble />
-      {data.total === 1 && (
+  const [showMoreFreqs, setShowMoreFreqs] = useState(false);
+  if (Object.keys(data.notifications).length === 0) {
+    const { hitSnps, sortedPopulations } = getSnpsInfo(data);
+    const coordinates = data.query_coordinates[0];
+    const allData = data["@graph"] || [];
+    const chromatinData = allData.filter((d) => d.method === "chromatin state");
+    const chipDataFilteredCount = filterOverlappingPeaks(
+      allData.filter((d) => d.method === "ChIP-seq")
+    )[1];
+    const dnaseDataFilteredCount = filterOverlappingPeaks(
+      allData.filter(
+        (d) =>
+          d.method === "FAIRE-seq" ||
+          d.method === "DNase-seq" ||
+          d.method === "ATAC-seq"
+      )
+    )[1];
```

Comment:
I have trouble with this part. You call `filterOverlappingPeaks` but you don’t use the resulting filtered datasets? You only use the count?

Reply:
Those data are used later in the component that I haven't transfer over. But I will add some more info in the data area.